combinations/hash.tsv

```diff
@@ -172,3 +172,5 @@ gatk=3.5,samtools=1.13
 requests=2.26	quay.io/bioconda/base-glibc-debian-bash:latest	0
 mirtop=0.4.24,samtools=1.13,r-base=4.1.1,r-data.table=1.14.2
 krakenuniq=0.5.8,pigz=2.6
+bioconda::gatk4=4.2.0.0,conda-forge::R=4.1,conda-forge::r-ggplot2=3.3.5,bioconda::datamash=1.1.0
```

Comment:
you don't need to specify the channel

Reply:
Mmm, I see! 

Thanks for the catch here @bgruening , now I realized that the `abc/xyz` is actually for docker images.

Reply:
```suggestion
gatk4=4.2.0.0,R=4.1,r-ggplot2=3.3.5,datamash=1.1.0
```